pkg/debug/spec.sh

```diff
@@ -142,6 +142,75 @@ add_pci_info() {
 __EOT__
 }
 
+get_modem_protocol() {
+    local sys_dev="$1"
+    local module="$(basename "$(readlink "${sys_dev}/device/driver/module")")"
```

Comment:
I think these yetus errors are reasonable to fix instead of ignore it...

Reply:
done, hopefully all yetus issues are fixed now